src/drt/test/fixture.cpp

```diff
@@ -544,6 +544,26 @@ void Fixture::makeKeepOutZoneRule(frLayerNum layer_num,
   design->getTech()->addUConstraint(std::move(con));
 }
 
+frLef58CutSpacingConstraint*
+Fixture::makeLef58CutSpacingConstraint_parallelOverlap(frLayerNum layer_num,
+                                                       frCoord spacing,
+                                                       bool parallelOverlap)
```

Comment:
why do you need bool parallelOverlap? I am assuming from the name of this function that it should only create PARALLELOVERLAP rules

Reply:
Also you don't need here to create an odb rule. Some rules require a pointer to the odb rule as they use that pointer directly without having to recreate all the rule data. However, this rule doesn't do that.